stea/__init__.py

```diff
@@ -67,20 +67,31 @@ def calculate(stea_input):
                                  stea_input.project_version,
                                  stea_input.config_date)
     request = SteaRequest(stea_input, project)
-
+  
     for profile_id, profile_data in stea_input.ecl_profiles.iteritems():
-        ecl_key = profile_data[SteaInputKeys.ECL_KEY]
-        start_year = profile_data.get(SteaInputKeys.START_YEAR)
-        end_year = profile_data.get(SteaInputKeys.END_YEAR)
-
-        request.add_ecl_profile(profile_id, ecl_key, first_year=start_year, last_year=end_year)
+        if not profile_id in project.profiles:
+          profile_list=[k for k,v in project.profiles.items() if ('Description' in v and v['Description']==profile_id)]
+        else:
+          profile_list=[profile_id]
+        if len(profile_list)>0:
+          ecl_key = profile_data[SteaInputKeys.ECL_KEY]
```

Comment:
You are using an indentation of two spaces - that is very non standard; please use four.